includes/controller/guest_login_controller.php

```diff
@@ -29,6 +29,8 @@ function guest_register() {
   $jabber = "";
   $hometown = "";
   $comment = "";
+  $native_lang = "";
+  $other_langs = "";
```

Comment:
These two variables are not initialized while showing the form, hence errors were displayed similar to 'Undefined variable $native_lang'